client/flutter/ios/Runner/Runner.entitlements

```diff
@@ -0,0 +1,8 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
Can you add a section in https://github.com/WorldHealthOrganization/app/blob/master/docs/BUILD.md that explains how to swap out this file for one that would work in production?

Reply:
Done.